frontend/public/locales/en/gcweb.json

```diff
@@ -87,6 +87,15 @@
     "page-message": "We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for.",
     "page-link": "Return to the <home>home page</home>"
   },
+  "public-not-found": {
+    "document-title": "Not found",
+    "page-title": "We couldn't find that web page",
+    "page-subtitle": "(Error 404)",
+    "page-message": "We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for.",
+    "unilingual-return-cdcp": "Return to the <cdcpLink>Canadian Dental Care Plan</cdcpLink> home page.",
+    "return-cdcp": "Return to the <englishCdcpLink>Canadian Dental Care Plan</EnglishCdcpLink> home page.",
```

Comment:
```suggestion
    "return-cdcp": "Return to the <englishCdcpLink>Canadian Dental Care Plan</englishCdcpLink> home page.",
```

Reply:
Capitalization corrected in commit eb69b75a6a9a85b7d1f4240106cb50e95b7d7853.